configuration/openmrs/i18n/registration/locale_fr.json

```diff
@@ -0,0 +1,28 @@
+{
+  "REGISTRATION_SEARCH_CONFIG_KEY": "<u>R</u>echerche",
+  "REGISTRATION_CREATE_NEW_CONFIG_KEY":"<u>C</u>réer nouveau",
+  "REGISTRATION_SEARCH_CONFIG_ACCESS_KEY": "e",
+  "REGISTRATION_CREATE_NEW_CONFIG_ACCESS_KEY": "n",
+  "REGISTRATION_VIEW_CONFIG_KEY": "Vue",
+  "REGISTRATION_PRINT_REG_CARD_LOCAL_KEY" : "Imprimer Reg . Carte(<u>L</u>Ocal)",
+  "REGISTRATION_PRINT_REG_CARD_KEY":"<u>P</u>rint carte d'enregistrement",
+  "REGISTRATION_PRINT_SUPPLEMENTAL_PAPER":"P<u>r</u>int papier supplémentaire",
+  "REGISTRATION_AGE_ERROR_KEY":"L'âge ne devrait être une valeur possitive",
+  "REGISTRATION_TELEPHONE_NUMBER_ERROR_KEY":"Numéro de téléphone doit avoir un minimum de 6 chiffres",
+  "REGISTRATION_CASTE_TEXT_ERROR_KEY":"Caste doit contenir que des caractères alphanumériques",
+  "REGISTRATION_BRING_CARD_TEXT_KEY":"Il est très important d'apporter la carte à l'hôpital.",
+  "REGISTRATION_TAKE_MEDICINES_TEXT_KEY": "Prendre des médicaments selon les instructions du médecin .",
+  "REGISTRATION_TIMINGS_TEXT_KEY":"Horaires d'inscription sont de 8 heures et demie-12h00 .",
+  "REGISTRATION_OPD_DAYS_TEXT_KEY" : "Jours OPD sont le lundi , mercredi, vendredi .",
+  "REGISTRATION_SUNDAYS_HOLIDAY_TEXT_KEY": "Les dimanches sont les jours fériés.",
+  "REGISTRATION_SMOKING_INJURIOUS_TEXT_KEY" : "Fumeurs , le tabac à chiquer peut causer le cancer , les maladies cardiaques et asthama .",
+  "REGISTRATION_INSTITUTE_NAME_TEXT_KEY": "Jan Swasthya Sahyog",
+  "REGISTRATION_INSTITUTE_REGISTERED_TEXT_KEY": "(Enregistré)",
+  "REGISTRATION_INSTITUTE_ADDRESS": "Ganiyari , District - Bilaspur",
+  "REGISTRATON_LATEST_KEY" : "Dernières",
+  "OBS_ABNORMAL_TRANSLATE_KEY":"Anormale",
+  "VITAL_SIGNS":"Signes vitaux",
+  "NUTRITIONAL_VALUES_LOCALE_KEY":"Valeurs nutritionnelles",
+  "TRIAGE_INFORMATION": "Informations de triage",
+  "FEE_INFORMATION_LOCALE_KEY":"Informations sur les frais"
```

Comment:
Does not sound right in French I think. Let's make this "Frais".

Reply:
[Done](https://github.com/mekomsolutions/bahmni-config-easydoc/pull/5/commits/95a2e975d58baf345f2e4434e608b27194205070)